cmd/fill/main.go

```
@@ -22,85 +24,43 @@ func main() {
 	ctx := context.Background()
 
 	conn, err := setupDatabase(ctx)
-	_ = conn
+	defer conn.Close(ctx)
```

golangci-lint (1.62.2)

SA5001: should check error returned from setupDatabase() before deferring conn.Close(ctx)

(staticcheck)